### DISCUSSION TITLE
【keyball39】ジャンパのはんだ付けに失敗してしまいました

### DISCUSSION THREAD

**Discussion by N7-tR0e3n** [Yowkees/keyball#703] [Q&A]
![IMG_1854](https://github.com/user-attachments/assets/0f85d3fe-454a-4d3e-8e49-69d14ad5a8f0)
ジャンパのはんだ付けに失敗し、パッドが剥がれてしまったものと思われます。
対処法をご教示のほどお願いします…

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

剥がれてしまったジャンパですが、下記画像の緑点２点を導通させれば、ジャンパーしたのと同じ回路になります。
下記緑点どうしをエナメル線やゴム被覆導線を使って接続してみてください。
![39jampfix](https://github.com/user-attachments/assets/8d1c3d6b-5242-4db6-859f-207c3d1a3245)

以上、お手数ですがご対応お願いいたします。

**Comment by N7-tR0e3n** [comment]
早急な対応ありがとうございました、試してみます。

**Comment by N7-tR0e3n** [comment]
ありがとうございます。上記の通り導通させたところ動作しました。
